parquet-hadoop/src/main/java/org/apache/parquet/hadoop/metadata/FileMetaData.java

```diff
@@ -50,16 +51,23 @@ public final class FileMetaData implements Serializable {
    * @throws NullPointerException if schema or keyValueMetaData is {@code null}
    */
   public FileMetaData(MessageType schema, Map<String, String> keyValueMetaData, String createdBy) {
-    this(schema, keyValueMetaData, createdBy, null);
+    this(schema, keyValueMetaData, createdBy, EncryptionType.UNENCRYPTED, null);
   }
-  
-  public FileMetaData(MessageType schema, Map<String, String> keyValueMetaData, String createdBy, InternalFileDecryptor fileDecryptor) {
+
+  public FileMetaData(MessageType schema, Map<String, String> keyValueMetaData, String createdBy,
+                      InternalFileDecryptor fileDecryptor) {
+    this(schema, keyValueMetaData, createdBy, null, fileDecryptor);
```

Comment:
```suggestion
    this(schema, keyValueMetaData, createdBy, EncryptionType.UNENCRYPTED, fileDecryptor);
```

Reply:
The encryption type is unknown here, but this method is not called anymore, so I'll deprecate it.